osu.Game/Overlays/Profile/Header/TopHeaderContainer.cs

```diff
@@ -51,124 +62,148 @@ private void load(OverlayColourProvider colourProvider)
-                                Padding = new MarginPadding { Left = 10 },
-                                Children = new Drawable[]
+                                new FillFlowContainer
                                 {
-                                    new FillFlowContainer
+                                    Direction = FillDirection.Horizontal,
+                                    Padding = new MarginPadding
                                     {
-                                        AutoSizeAxes = Axes.Both,
-                                        Direction = FillDirection.Vertical,
-                                        Children = new Drawable[]
-                                        {
-                                            new FillFlowContainer
-                                            {
-                                                AutoSizeAxes = Axes.Both,
-                                                Direction = FillDirection.Horizontal,
-                                                Spacing = new Vector2(5),
-                                                Children = new Drawable[]
-                                                {
-                                                    usernameText = new OsuSpriteText
-                                                    {
-                                                        Font = OsuFont.GetFont(size: 24, weight: FontWeight.Regular)
-                                                    },
-                                                    openUserExternally = new ExternalLinkButton
-                                                    {
-                                                        Anchor = Anchor.CentreLeft,
-                                                        Origin = Anchor.CentreLeft,
-                                                    },
-                                                    groupBadgeFlow = new GroupBadgeFlow
-                                                    {
-                                                        Anchor = Anchor.CentreLeft,
-                                                        Origin = Anchor.CentreLeft,
-                                                    }
-                                                }
-                                            },
-                                            titleText = new OsuSpriteText
-                                            {
-                                                Font = OsuFont.GetFont(size: 18, weight: FontWeight.Regular)
-                                            },
-                                        }
+                                        Left = UserProfileOverlay.CONTENT_X_MARGIN,
+                                        Vertical = vertical_padding
                                     },
-                                    new FillFlowContainer
+                                    Spacing = new Vector2(20, 0),
```

Comment:
Spacing changes to 10 when cover is collapsed.
![image](https://user-images.githubusercontent.com/35318437/214457949-54ff3f3c-7170-44a0-b071-7f0b99943561.png)

![image](https://user-images.githubusercontent.com/35318437/214457926-1dbddeb4-986c-43a8-b475-44ddb44a16aa.png)

Reply:
Have applied a fix for this, good eye.